config/application.rb

```diff
@@ -15,6 +15,9 @@
 require "failbot_rails"
 FailbotRails.setup("classroom#{'-staging' if Rails.env.staging?}")
 
+# configure chewy
+require_relative "chewy"
```

Comment:
I'm not an expert on autoloading, would it be appropriate to move the file somewhere else so that it is autoloaded?

Reply:
Thats a good question. @d12 @tarebyte would either of you know?

Reply:
@BenEmdon could we just move this to `config/initializers`?